scriptmodules/supplementary/skyscraper.sh

```diff
@@ -25,6 +25,7 @@ function depends_skyscraper() {
 
 function sources_skyscraper() {
     gitPullOrClone
+    echo VERSION=\"$(_get_branch_skyscraper)\" > VERSION
```

Comment:
what's the point of the `VERSION` file ? Is it really needed ?

Reply:
> what's the point of the `VERSION` file ? Is it really needed ?

It is part of the repo anyway. I guess this line is a leftover, can't remember why I put it there.